packages/metal-clay-button/src/MetalClayButton.soy

```diff
@@ -18,12 +18,12 @@
 
 
 	{if $href}
-		<a href="{$href}" {call .attributes data="all" /} aria-label="{$aria_description}">
+		<a href="{$href}" {call .attributes data="all" /} aria-label="{$ariaLabel}">
```

Comment:
testMetalClayButtonWithAriaDescription.html file is not necessary anymore. Right?